### PR TITLE
Integ

### PR DESCRIPTION
**Pull request by francoisminaud**
PR to import Pypi and Elasticsearch-logging charts

### CODE REVIEW

**Comment by francoisminaud** [comment]
Can you please review it @desaintmartin ?

File: charts/pypi/values.yaml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  # hosts:
+  #   - upload.myhost.com
+  # tls:
+  #   - secretName: upload.myhost.com-tls
+  #     hosts:
+  #       - upload.myhost.com
+
+basicAuth:
+  # Authenticate for uploading, use existing secret
+  secretName: "basic-auth-upload-htpasswd"
```
Comment (desaintmartin) [inline charts/pypi/values.yaml:55]: How can it pass the test?

**Review by desaintmartin — CHANGES_REQUESTED** [review]
Maybe you should add to .gitignore the .tar.gz files has well, from my understanding chart-releaser will do `helm dep build` before `helm package`.

File: charts/pypi/values.yaml (hunk 1 above)
Comment (francoisminaud) [reply charts/pypi/values.yaml]: It has been reworked : https://github.com/wiremind/wiremind-helm-charts/commit/30b3a5ff4c6414ccb3361aee7cdabf3d1a0a9a06#diff-3501b229ecff8a86900788c6eaf849bd7c8c88df561e718e83b766f5189b04d8

File: charts/elasticsearch-logging/requirements.lock [hunk 2]
```diff
@@ -0,0 +1,21 @@
+dependencies:
```
[the remark this answers is not in the document]
Comment (francoisminaud) [reply charts/elasticsearch-logging/requirements.lock]: It was a mistake on my side, it has been fixed.

File: charts/elasticsearch-logging/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+appVersion: 7.9.2
+dependencies:
+  - name: elasticsearch
+    repository: "https://helm.elastic.co"
+    version: "7.10.0"
+    alias: es-data-hot
+#  - name: elasticsearch
+#    repository: "https://helm.elastic.co"
+#    version: "*"
+#    alias: es-data-warm
```
[the remark this answers is not in the document]
Comment (francoisminaud) [reply charts/elasticsearch-logging/Chart.yaml]: This is done, conditions were added for hot / warm / cold.

File: charts/clair/templates/ingress.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+{{- if .Values.ingress.enabled -}}
+{{- $serviceName := include "fullname" . -}}
+{{- $servicePort := .Values.service.externalApiPort -}}
+apiVersion: extensions/v1beta1
```
[the remark this answers is not in the document]
Comment (francoisminaud) [reply charts/clair/templates/ingress.yaml]: This piece of code was masked because disabled.
It has been updated though.

**Pull request merged**